cmd/ctl/pkg/renew/renew.go

```diff
@@ -112,6 +112,10 @@ func (o *Options) Validate(cmd *cobra.Command, args []string) error {
 		return errors.New("cannot specify --namespace flag in conjunction with --all flag")
 	}
 
+	if !o.All && len(args) == 0 {
+		return errors.New("inform one or more Certificate resource name or use the --all flag to renew all Certificate resources")
```

Comment:
"inform" is not the right word, but my original suggestion was also not quite right. Here's my amended suggestion:

```suggestion
		return errors.New("please supply one or more Certificate resource names or use the --all flag to renew all Certificate resources")
```